test/Fencer/Rules/Test.hs

```diff
@@ -26,20 +26,38 @@ tests = testGroup "Rule tests"
   [ test_rulesLoadRulesYaml
   , test_rulesLoadRulesNonYaml
   , test_rulesLoadRulesRecursively
+  , test_rulesLoadRulesDotDirectory
   ]
 
+-- | A helper function for loading rules and making sure they are as
+-- expected.
+loadRules :: String -> Bool -> IO ()
```

Comment:
`String` and `Bool` should be named arguments.

Reply:
Also:

```suggestion
loadRules :: String -> Bool -> Assertion
```

Reply:
Also, what about generalizing it a bit?

```haskell
-- | Create given directory structure and check that 'loadRulesFromDirectory'
-- produces expected result.
expectLoadRules
  :: "ignoreDotFiles" :! Bool
  -> "files" :! [(FilePath, Text)]
  -> "result" :! [DomainDefinition]
  -> Assertion
```

Reply:
This looks good! I'll adopt it.

Reply:
Done.